pydantic/_internal/_config.py

```diff
@@ -116,6 +116,13 @@ def for_model(cls, bases: tuple[type[Any], ...], namespace: dict[str, Any], kwar
         config_class_from_namespace = namespace.get('Config')
         config_dict_from_namespace = namespace.get('model_config')
 
+        raw_annotations = namespace.get('__annotations__', {})
+        if raw_annotations.get('model_config') and not config_dict_from_namespace:
+            raise PydanticUserError(
+                '"model_config" cannot be used as a model property. Use "model_config" for model configuration',
```

Comment:
same suggestion here, let's use `model field` instead of `model property`

Reply:
used `model field`